pkg/ddl/table_split_test.go

```diff
@@ -45,7 +45,7 @@ func TestTableSplit(t *testing.T) {
 	tk := testkit.NewTestKit(t, store)
 	tk.MustExec("use test")
 	// Synced split table region.
-	tk.MustExec("set global tidb_scatter_region = 1")
+	tk.MustExec("set @@session.tidb_scatter_region = 1")
```

Comment:
Need also keep set global level sys var testcase, that means we need add session level test case and keep global level tests.

Reply:
In this case, the reason for setting `set @@session.tidb_scatter_region = 1` is waiting split region finish before create table statement return. However, we do indeed need to add a test case at the global level.